### ISSUE TITLE
I know this is the incorrect place to ask this

### ISSUE THREAD

**Issue by gravypod** [schlae/crt-driver#1]
Hey, I recently found your youtube channel. I've been looking for some way to contact you to ask you about this but I couldn't find a way to contact you on tube time. I've got to say, from what I've seen so far I'm a huge fan of your work!

I was wondering, do blue CRTs exist? I've heard people talk about blue phosphorous but I have never been able to find a place to source them. 

I'd love to be able to use a blue CRT for a future project. 

Thanks for your time and sorry for contacting you via this (mean) method!

**Comment by schlae** [comment]
Yes, you will want a CRT with a P11 phosphor (see [the list of phosphor types](http://www.bunkerofdoom.com/tubes/crt/crt_phosphor_research.pdf)). There are types other than P11 that are blue, but it is the most common. Something like a 3BP11 would work. Check Ebay!

**Comment by WeisTekEng** [comment]
Heh, guess I'll jump in on this as well. 

Will this circuit work for a 1 inch view finder tube? Thanks and great work.
